### ISSUE TITLE
'View in Room' guard

### ISSUE THREAD

**Issue by dzucconi** [artsy/metaphysics#175]
https://www.artsy.net/artwork/giorgio-griffa-sezione-aurea-obliquo-finale-628

When photographs have surrounding install context the in room view is disfigured. In this instance, it dramatically overflows off the 'wall'.

We should disable the `is_hangable` field if the aspect ratio of the primary image diverges too much from the aspect ratio of the artwork measurements.

**Comment by alloy** [comment]
[MP](https://github.com/artsy/metaphysics/blob/4e15e5910486d907f1c2bcb00b39dd725a1d3c89/schema/artwork/utilities.js) and [Eigen](https://github.com/artsy/eigen/blob/ac948a350184f9fd64aa5c6217e5f6f8119b0375/Artsy/Models/API_Models/Partner_Metadata/Artwork.m#L270-L273) both have a bunch of heuristics to determine whether or not these works can be shown in a ‘view in room’ scene. I wonder if it isn’t time to consolidate those into Gravity.

@joeyAghion any thoughts on that?

**Comment by joeyAghion** [comment]
I think it would be great to promote this logic to the API, assuming there's shared meaning to agree on. We probably can't completely eliminate the client logic, since different clients may have different concepts of what's "too big." (Also I don't want to step into anything controversial w.r.t. non-flat hanging pieces or sculptures that can be "viewed in room.")

**Comment by alloy** [comment]
Agreed, I’m thinking we just take extract the intersection of the heuristics that are being used in both places right now. I think this specific case that the ticket was about would apply to both as well, but that’s definitely something to check.
